<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>reg</class>
 <widget class="QWidget" name="reg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>注册</string>
  </property>
  <property name="windowIcon">
   <iconset resource="pic.qrc">
    <normaloff>:/app_icon.ico</normaloff>:/app_icon.ico</iconset>
  </property>
  <widget class="QWidget" name="horizontalWidget" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>账       号</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="le_account"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalWidget_2" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>密       码</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="le_password"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalWidget_3" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>确认密码</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="le_pasw_c"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="bt_finish">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>240</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>完成</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_exit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>240</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>-11</x>
     <y>-10</y>
     <width>441</width>
     <height>331</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/bg1.jpg);</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-1</y>
     <width>411</width>
     <height>301</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 100);</string>
   </property>
  </widget>
  <zorder>widget</zorder>
  <zorder>widget_2</zorder>
  <zorder>bt_exit</zorder>
  <zorder>bt_finish</zorder>
  <zorder>horizontalWidget</zorder>
  <zorder>horizontalWidget_2</zorder>
  <zorder>horizontalWidget_3</zorder>
 </widget>
 <resources>
  <include location="pic.qrc"/>
 </resources>
 <connections/>
</ui>
